src/Backend/Modules/Location/Command/CopyLocationWidgetsToOtherLocaleHandler.php

```diff
@@ -2,32 +2,27 @@
 
 namespace Backend\Modules\Location\Command;
 
-use Common\ModuleExtraType;
+use Backend\Modules\Pages\Domain\ModuleExtra\ModuleExtra;
+use Backend\Modules\Pages\Domain\ModuleExtra\ModuleExtraRepository;
 use SpoonDatabase;
 
 final class CopyLocationWidgetsToOtherLocaleHandler
 {
-    /** @var SpoonDatabase */
-    private $database;
+    /** @var ModuleExtraRepository */
+    private $moduleExtraRepository;
 
-    public function __construct(SpoonDatabase $database)
+    public function __construct(ModuleExtraRepository $moduleExtraRepository)
     {
-        $this->database = $database;
+        $this->moduleExtraRepository = $moduleExtraRepository;
     }
 
     public function handle(CopyLocationWidgetsToOtherLocale $copyLocationWidgetsToOtherLocale): void
     {
-        $currentWidgets = $this->database->getRecords(
-            'SELECT * FROM modules_extras WHERE module = ? AND type = ? AND action = ?',
-            [
-                'Location',
-                ModuleExtraType::widget(),
-                'Location'
-            ]
-        );
+        $currentWidgets = $this->moduleExtraRepository->findWidgetsByModuleAndAction('Location', 'Location');
 
+        /** @var ModuleExtra $currentWidget */
         foreach ($currentWidgets as $currentWidget) {
```

Comment:
Add the typehint on the return of the method. Not here since you will need to add it everywhere you use it